### PR TITLE
style: run prettier on web/api/m-r

### PR DESCRIPTION
**Pull request by nschonni**
Not run on code fences

### CODE REVIEW

File: files/en-us/web/api/mouseevent/mouseevent/index.md [hunk 1]
```diff
@@ -51,21 +53,23 @@ new MouseEvent(type, options)
     - `metaKey` {{optional_inline}}
       - : A boolean value indicating if the <kbd>meta</kbd> key was simultaneously pressed. It defaults to `false`.
     - `button` {{optional_inline}}
+
       - : A number, defaulting to `0`, that describes which button is pressed during events related to the press or release of a button:
-        | Value | Meaning                                                         |
+        | Value | Meaning |
```
Comment (teoli2003) [inline files/en-us/web/api/mouseevent/mouseevent/index.md:58]: This weird. I had the impression that table length got aligned by Prettier in other cases, but this is doing exactly the opposite. Did I miss something here? (or before)

Comment (nschonni) [reply files/en-us/web/api/mouseevent/mouseevent/index.md:58]: Not sure, but maybe because it's inside a list an the others aren't

**Review by teoli2003 — CHANGES_REQUESTED** [review]
The only thing that is strange here is that table. I would like to understand what's going on with it.

File: files/en-us/web/api/mouseevent/mouseevent/index.md (hunk 1 above)
Comment (nschonni) [reply files/en-us/web/api/mouseevent/mouseevent/index.md]: Yeah, looks like adding the space and re-running it resolved the formatting

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**